docs/test_runners.md

```diff
@@ -0,0 +1,42 @@
+# Elección de `test runner`
+> Objetivo 4
+
+## Criterios
+Es de principal importancia que la decisión se tome teniendo en cuenta los principios F.I.R.S.T.
+* Fast
+* Independent
+* Repeatable
+* Self-validating
+* Timely
+
+Así que todas las opciones que se tendrán en cuentra cumplirán como mínimo con estás características.
+
+## Opciones
+> Tras leer en varios foros y teniendo en cuenta las características del modelo, las siguientes son las opciones que tendremos en cosideración pese a que haya muchas alternativas más.
+
```

Comment:
No hay ningún criterio a tener en cuenta. Lo principal es que enunciéis los criterios.

Reply:
Los criterios están en la sección anterior: `Facilidad de uso y aprendizaje, facilidad de instalación, velocidad.`

¿Es que no son válidos?

Reply:
¿Facilidad de aprendizaje es un criterio objetivo? ¿O es una excusa para que elijas el que más te gusta?